include/mesh.h

```diff
@@ -184,6 +192,8 @@ class Mesh {
  private:
   // Locate a particle in mesh cells
   bool locate_particle_cells(std::shared_ptr<mpm::ParticleBase<Tdim>> particle);
+
+ protected:
```

Comment:
Why `protected`?

Reply:
I am sorry. Not sure when I did this... Might be merging issue